### ISSUE TITLE
very fragmented draft genome

### ISSUE THREAD

**Issue by sqwwww** [theaidenlab/3d-dna#171]
Hi,
I'm trying to scaffolding a draft for which the contigN50 is 8k. It's a very fragmented draft with more than 800 million contigs in it. I ran 3d-dna with default parameters, and put the output image in the attachment. Nearly half of the draft didn't assembled.

I have checked the parameters mentioned in DNA zoo, and I know I should adjust the -i smaller, I will try to run 3d-dna with -i 5000.

Is there any other parameters I should also pay attention to for the very fragmented draft? could you give me some suggestions?
![a9b47cb8b587cb260b3bda63123089b](https://github.com/aidenlab/3d-dna/assets/98629662/fdca54f0-34ea-4ecf-a5b7-e461faa16964)

**Comment by SCD-cloud** [comment]
hello,have you solved your problem?i have the same issue.

**Comment by sqwwww** [comment]
> hello,have you solved your problem?i have the same issue.

No, I can't scaffold a good quality genome with Illumina DNA reads and Hi-C data, probably because of highly repetitive sequences in the genome. Finally, I sequenced some PacBio long reads to assemble the genome.
